setup.py

```diff
@@ -31,7 +31,13 @@
 
 install_requires = [
     'amqp>=1.4.9,<2.0',
+    'Flask>=0.11.1',
+    'Flask-Admin>=1.4.2',
+    'Flask-Breadcrumbs>=0.3.0',
+    'Flask-Script>=2.0.5',
+    'Flask-CLI>=0.4.0',
```

Comment:
Yes, in fact I resubmitted this patch without a dependency on Flask-CLI: https://github.com/inspirehep/inspire-next/pull/1319#issue-165122304. Our `setup.py` is without doubt full of scar tissue.